.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+*.ps1
```

Comment:
This would ignore all powershell files - is this exactly what you want?

Reply:
Yes, for the time being.

Reply:
Cool 👍